androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/splash/SplashScreen.kt

```diff
@@ -1,78 +1,260 @@
             )
+
+            val gradient = Brush.verticalGradient(
+                colors = listOf(
+                    Black.copy(alpha = animateAlpha * InitialTopGradientAlpha),
+                    Black.copy(
+                        alpha = (animateAlpha * BottomGradientAlphaMultiplier) +
+                            InitialBottomGradientAlpha
+                    )
+                )
+            )
+            Box(
+                modifier = Modifier
+                    .matchParentSize()
+                    .background(gradient)
+            )
+
+            AnimatedVisibility(
+                visible = shouldShowLogo,
+                enter = fadeIn(animationSpec = tween(LogoDurationInMillis))
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.logo_white),
+                    contentDescription = null,
+                    modifier = Modifier
+                        .offset(animateLogoOffset.x.dp, animateLogoOffset.y.dp)
+                        .scale(animateLogoScale)
+                )
+            }
         }
     }
 }
 
+@Composable
+private fun LoginForm(
+    email: String,
+    password: String,
+    onEmailChange: (String) -> Unit,
+    onPasswordChange: (String) -> Unit,
+    onLogInClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        verticalArrangement = Arrangement.spacedBy(20.dp),
+        modifier = modifier
+            .fillMaxSize()
+            .wrapContentHeight()
+            .padding(horizontal = 24.dp)
+    ) {
+        PrimaryTextField(
+            value = email,
+            onValueChange = onEmailChange,
+            placeholder = stringResource(id = R.string.login_email),
+            keyboardType = KeyboardType.Email,
+            contentDescription = LoginEmailFieldContentDescription
+        )
+        Box {
+            PrimaryTextField(
+                value = password,
+                onValueChange = onPasswordChange,
+                placeholder = stringResource(id = R.string.login_password),
+                visualTransformation = PasswordVisualTransformation(),
+                imeAction = ImeAction.Done,
+                contentDescription = LoginPasswordFieldContentDescription
+            )
+            Text(
+                text = stringResource(id = R.string.login_forgot),
+                color = MaterialTheme.colors.onSurface.copy(alpha = ForgotTextAlpha),
+                style = MaterialTheme.typography.body2,
+                modifier = Modifier
+                    .wrapContentHeight()
+                    .align(Alignment.CenterEnd)
+                    .padding(end = 18.dp)
+            )
+        }
+        PrimaryButton(
+            text = stringResource(id = R.string.login_button),
+            onClick = onLogInClick,
+            modifier = Modifier
+                .fillMaxWidth()
+                .semantics { contentDescription = LoginButtonContentDescription },
```

Comment:
`contentDescription` for this is not necessarily needed if you want to use it for integration testing. We can find with text and click on the component.
```suggestion
```